### ISSUE TITLE
wasted space bellow bottom nav bar

### ISSUE THREAD

**Issue by doudoud82** [mihonapp/mihon#1330]
### Steps to reproduce

Update to the latest mihon beta release after r6916.

### Expected behavior

expecting the app to look like this:
![Screenshot_20241015-193557_Mihon](https://github.com/user-attachments/assets/a19d3e91-b897-4eab-b30d-30fa6a5aec61)
![Screenshot_20241015-193549_Mihon](https://github.com/user-attachments/assets/e12240a4-b234-4a92-b3cc-1d0648beea2b)

### Actual behavior

the app looks like this:
![Screenshot_20241015-193015_Mihon](https://github.com/user-attachments/assets/bcc24f70-7695-4b60-949b-2838b300a70d)
![Screenshot_20241015-192954_Mihon](https://github.com/user-attachments/assets/1a451226-efdd-41e5-a479-5122f85b55f7)
![Screenshot_20241015-193001_Mihon](https://github.com/user-attachments/assets/c44f09e9-321f-43c2-aae3-454ff25e1797)

### Mihon version

Beta r6938

### Android version

Android 8

### Device

Samsung S7

### Other details

I noticed this today when updating tachiyomiSY preview build r566 and got advised to report it to mihon. upon further investigation, I found out that  Mihon Preview r6916 doesn't exhibit this behaviour whereas newer builds do.
Also when reading a chapter this wasted space doesn't appear and the screen is used fully.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have gone through the [FAQ](https://mihon.app/docs/faq/general) and [troubleshooting guide](https://mihon.app/docs/guides/troubleshooting/).
- [X] I have updated the app to version **[0.16.5](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by MajorTanya** [comment]
Just to be clear, you _were_ using r6916 and then _updated to_ latest, r6938?

**Comment by doudoud82** [comment]
> Just to be clear, you _were_ using r6916 and then _updated to_ latest, r6938? Did you also observe this in the intermediate r6926?
I am using TachiyomiSy mainly in today's update I noticed this and when talked about it in Discord I got recommended to report the issue to Mihon.
I decided to check if the problem is in Mihon too(otherwise it is not worth reporting here) and I found that the build r6916 doesn't have this problem while any newer build after it does have it. in other words, r6926 also has this behaviour; and r6916 is the oldest build I tested that didn't exhibit it. I did this to potentially help identify which change caused it.

**Comment by MajorTanya** [comment]
> > Just to be clear, you _were_ using r6916 and then _updated to_ latest, r6938? Did you also observe this in the intermediate r6926?
> 
> I am using TachiyomiSy mainly in today's update I noticed this and when talked about it in Discord I got recommended to report the issue to Mihon. I decided to check if the problem is in Mihon too(otherwise it is not worth reporting here) and I found that the build r6916 doesn't have this problem while any newer build after it does have it. in other words, r6926 also has this behaviour; and r6916 is the oldest build I tested that didn't exhibit it. I did this to potentially help identify which change caused it.

Thanks, that'll be helpful in narrowing it down 👍

**Comment by MajorTanya** [comment]
Could you tap the "Version" field in the More view and paste the copied data here?

**Comment by doudoud82** [comment]
> Could you tap the "Version" field in the More view and paste the copied data here?

No problem.
App version: 0.16.5-6938 (standard, 2e2c8d36c, 7, 2024-10-14T23:03:33Z)
Android version: 8.0.0 (SDK 26; build ALEXNDR.G930FXXU8EUE1)
Device brand: samsung
Device manufacturer: samsung
Device name: herolte (heroltexx)
Device model: SM-G930F
WebView: Chrome 129.0.6668.100

**Comment by MajorTanya** [comment]
Okay, so.
I checked the "good" and "bad" versions in an Android 8.0.0 emulator with the screen dimensions set to whatever the S7 has. I cannot replicate this problem.

It also doesn't seem to exist on more current devices, both in emulation as well as actual installs.

I'm not sure why it appears for you. Some remaining ideas:
- Did you try force closing the app and restarting it?
- Did you try to change the theme to something else? If yes, and the issue persists, please provide one (1) screenshot of _only the bad version_ and please Do Not use the Amoled/Pure Black mode for that
- Did you try uninstalling Mihon and reinstalling it? Maybe something went weird during the update/install process?

**Comment by doudoud82** [comment]
> * Did you try force closing the app and restarting it?

no, I mainly use TachiSy as I said previously and this behaviour is happening on a fresh install of mihon preview.
> * Did you try to change the theme to something else? If yes, and the issue persists, please provide one (1) screenshot of _only the bad version_ and please Do Not use the Amoled/Pure Black mode for that

Here is a screenshot for when disabling pure black.
![Screenshot_20241015-210130_Mihon](https://github.com/user-attachments/assets/88c9470b-dcc2-4131-a8d2-4d6a163a50a1)
> * Did you try uninstalling Mihon and reinstalling it? Maybe something went weird during the update/install process?

I reinstalled it a couple of times since I switched between multiple versions(mainly r6938 r6926 r6916) to check from which version this started to happen.

**Comment by MajorTanya** [comment]
Very weird. As I said, I can't reproduce this in an emulator and I don't have a real S7 at hand. I'll leave the issue open in case someone else figures something out.

**Comment by doudoud82** [comment]
understandable unless it is possible to replicate the behaviour it will be impossible to attempt to fix something that cannot be replicated to test and such.
this video shows the 3 versions I redownload and reinstalled.
https://github.com/user-attachments/assets/f36ad85d-34f3-4892-beef-c085e68a43ff

either way, I guess this will be on hold till more info or it can be reproduced since it doesn't affect when reading chapters only UI.
Have a nice day.

**Comment by doudoud82** [comment]
@nobSEIFO rather interisting didn't notice this behavior before, thanks for the info.

**Comment by MajorTanya** [comment]
This is most likely down to us removing a deprecated UI library that handled displays of older Android versions differently. We didn't increase the SDK version or anything like that (people with Android 8 wouldn't be able to install it at all in that case).

**Comment by nobSEIFO** [comment]
@MajorTanya is there any fix planned for this issue , or it will be left as it is

**Comment by MajorTanya** [comment]
> @MajorTanya is there any fix planned for this issue , or it will be left as it is

Don't comment unless you have something to contribute to the issue, please. We'll find a fix when we find it.

**Comment by BrutuZ** [comment]
> Would adding the line `<item name="android:windowFullscreen">true</item>` in line 9 of [this](https://github.com/mihonapp/mihon/blob/main/app/src/main/res/values/themes.xml) potentially solve this?

I'm guessing that would hide the system status bar too (notifications, clock, battery), which is not desirable outside of the reader given the many features that rely on notifications during usage.

**Comment by BrutuZ** [comment]
> what features rely on notifications only

App updates

**Comment by dajotim937** [comment]
Add to statistics: Android 9 / Nokia 8, I have same empty space bellow bottom nav bar (noticed in komikku and checked with latest stable mihon 0.19.4).

This empty space is not only bellow bottom navbar. It also empty when you open manga list in any extension. You can see display of the list is "cropped" at the bottom. Same goes for settings, when you scroll you can see it also has empty space at the end of the screen.

Also, this empty space will disappear if you open reader (open any chapter) and then close reader/chapter. And it will not appear until restart of app.
